### ISSUE TITLE
Split package

### ISSUE THREAD

**Issue by tuvistavie** [wende/autocomplete-elixir#36]
In recent versions, this package has integrated UI features (do/end highlighting).
Given the philosophy of Atom, I think it would be better to keep
this package as an autocompletion provider, and move other features
unrelated with autocompletion in their own package.
Any thoughts?

**Comment by wende** [comment]
Agreed. Because my intention was to build complete elixir environment I will leave it as it is right now. I've added option to disable highlighting for those who don't like it.
When I add more of the functionality I will start to think about spliting it to different packages.

**Comment by tuvistavie** [comment]
Great, thanks for the follow up!
